docs/integrations/sources/surveycto.md

```diff
@@ -0,0 +1,42 @@
+# SurveyCTO
+
+This page guides you through the process of setting up the SurveyCTO source connector.
+
+## Prerequisites
+
+- Server Name `The name of the ServerCTO server`
+- Your SurveCTO `Username`
+- Your SurveyCTO `Password`
+- Form ID `Unique Identifier for one of your forms`
+- Start Date `Start Date default`
+
+## Set up the SurveyCTO source connection
```

Comment:
This is not a simple setup, I think people will appreciate a few details on how to correctly setup the SurveyCTO account, it requires:

- create the account
- create your form
- publish your form
- give your user an API consumer permission to the existing role or create a user with that role and permission.